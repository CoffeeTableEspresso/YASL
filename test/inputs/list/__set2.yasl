
const x = [ '1', '2', '3']

echo x

x[0]  = 'first'

echo x
