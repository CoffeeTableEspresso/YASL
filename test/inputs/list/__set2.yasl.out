[1, 2, 3]
[first, 2, 3]
